site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -155,6 +155,59 @@
             </div>
 
 
+            <div id="manual-grading" class="customization_item">
+                <h2>Manual Grading</h2>
+                <p class="rg_info_message">You may manually assign final grades to specific students. The note is only visible to the instructor, and it will only appear if instructor_notes is enabled under Display.</p>
+                <input type="hidden" name="csrf_token" value="{{ csrfToken }}" />
+                <input type="hidden" name="option" value="-1" />
+                <div class="option">
+                    <label for="manual_grading_user_id">
+                        <span>USER ID:</span>
+                        <input class="option-input" type="text" name="manual_grading_user_id" id="manual_grading_user_id">
+                    </label>
+<label for="manual_grading_grade">
+    <span>Final Grade:</span>
+    <select name="manual_grading_grade" id="manual_grading_grade">
+        <option disabled selected value> -- select an option -- </option>
+        {% set grades = ["A", "A-", "B+", "B", "B-", "C+", "C", "C-", "D+", "D", "F"] %}
+        {% for grade in grades %}
+            <option value="{{ grade }}">{{ grade }}</option>
+        {% endfor %}
+    </select>
+</label>
+
+                    <label for="manual_grading_note">
```

Comment:
The indentation is a bit off here.

Reply:
[dbd3009](https://github.com/Submitty/Submitty/pull/10653/commits/dbd30092ef46a3ebf32207881d9ee672ab0c5f72)